foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/generator/SparkApplicationGenerator.java

```diff
@@ -143,6 +145,21 @@ public void generate(GenerationContext context) {
         vc.put(VelocityProperty.PIPELINE, pipeline.getName());
         vc.put(VelocityProperty.DOCKER_PROJECT_REPOSITORY_URL, dockerProjectRepositoryUrl);
 
+        /*
+         * Autogenerate the environment variables when a filestore is declared for the pipeline
+         */
+        List<FileStore> fileStores = pipeline.getFileStores();
+        boolean hasFileStore = fileStores.isEmpty();
```

Comment:
S: This is a very misleading variable name. I would invert either the name or the assignment. Right now it reads as if we only iterate over the decorated file stores if there are no file stores.

Reply:
Refactored.